index.html

```diff
@@ -1,14 +1,17 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
     <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <link href="style.css" rel="stylesheet" />
+        <title>Microverse Second Trial</title>
     </head>
     <body>
         <div id="main">
             <h1 id="title">Dr. Norman Borlaug</h1>
             <h4 id="sub-title">The man who saved a billion lives</h4>
             <div id="img-div">
```

Comment:
`div` tag does not convey any meaning, instead, you could use the `section` tag which refers to a group of items in this case a hero section.